packages/react/src/FormattingToolbar/components/DefaultButtons/CreateLinkButton.tsx

```diff
@@ -0,0 +1,43 @@
+import { useCallback } from "react";
+import { BlockNoteEditor } from "@blocknote/core";
+import { RiLink } from "react-icons/ri";
+import LinkToolbarButton from "../LinkToolbarButton";
+import { formatKeyboardShortcut } from "../../../utils";
+
+export const CreateLinkButton = (props: { editor: BlockNoteEditor }) => {
+  const setLink = useCallback(
```

Comment:
I don't think useCallback is useful in this function, especially with [props] as parameter.

The reason is that most functions are invoked immediately anyway (e.g. `linkIsActive()`)